en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -326,4 +352,104 @@ To manage the error in the Error Store Explorer, follow the procedure below:
+
+    ```
+    mysql> select * from SweetProductionTable;
+    +---------+--------+-----------+
+    | batchID | amount | factoryID |
+    +---------+--------+-----------+
+    |       1 |   45.6 |       102 |
+    +---------+--------+-----------+
+    1 row in set (0.00 sec)
+    ```
+
+Use the same curl command as above to publish another event. However, this time our event will fail since the batchID 
+column which is defined as the primary key has a duplicate value of 1 in the second message. You will see the following 
+log in the wso2si terminal.
+
+    ```
+    [2020-11-03 11:48:28,951] ERROR {io.siddhi.core.table.Table} - Error on 'StoreRdbmsError' while performing add for events  at 'SweetProductionTable'. Events saved 'EventChunk{first=StreamEvent{ timestamp=1604384308930, beforeWindowData=null, onAfterWindowData=null, outputData=[1, 45.6, 102], type=CURRENT, next=null}}'
+    ```
+#### Step 7.3: Manage the error in the Error Store Explorer
+To manage the error in the Error Store Explorer, follow the procedure below:
+    
+1. To open the Error Store Explorer, open Streaming Integrator Tooling, click **Tools** and then click 
+**Error Store Explorer**.
+
+    ![Access Error Store](../../images/handling-requests-with-errors/access-error-store-explorer.png)
+    
+2. In the **Siddhi app** field, select **StoreRdbmsError** Siddhi application and then click **Fetch**.
+    
+    As a result, an error is displayed as follows.
+    
+    ![Error Store Explorer](../../images/handling-requests-with-errors/error-store-explorer-with-StoreRdbmsError.png)
+    
+    This indicates that the event was dropped because the end point was not available.
+    
+2. To view details of the error, click **Detailed Info**. The following is displayed.
+
+    ![Error Entry](../../images/handling-requests-with-errors/error-entry-for-StoreRdbmsError.png)
+    
+    The erroneous event is displayed as a editable table as shown below. Change to value of batchID from 1 to 2 and 
+    click replay.
+    
+    ![Replay Error](../../images/handling-requests-with-errors/replay-error-StoreRdbmsError.png)
+    
+    As a result, the **Error Entry** dialog box closes, and the **Error Store Explorer** dialog box is displayed with 
+    no errors.
+    
+    You can select all from `SweetProductionTable` to verify if the edited event has been added successfuly.
+    ```
+        mysql> select * from SweetProductionTable;
+        +---------+--------+-----------+
+        | batchID | amount | factoryID |
+        +---------+--------+-----------+
+        |       1 |   45.6 |       102 |
+        |       2 |   45.6 |       102 |
+        +---------+--------+-----------+
+        2 rows in set (0.00 sec)
+    ```
+
+Similarly, other databases errors such as foreign key violation errors, not null errors etc. and even database 
+connection errors that can occur if the database goes down when an event is sent will be caught and saved in the Error 
+Store for replaying.
```

Comment:
Shall we break this into two sentences and say
1. Other database errors such as foreign key violation errors, not null errors etc.
2. Connection errors that occur if the database goes down

Reply:
Fixed in a5ef3f0